futurex_openedx_extensions/dashboard/views.py

```diff
@@ -292,6 +293,34 @@ def get(self, request: Any, username: str, *args: Any, **kwargs: Any) -> JsonRes
         )
 
 
+class DataExportTasksView(
+    ListAPIView, UpdateAPIView, FXViewRoleInfoMixin
+):  # pylint: disable=too-many-ancestors
+    """View to get the list of learners for a course"""
+    serializer_class = serializers.DataExportTaskSerializer
+    permission_classes = [FXHasTenantCourseAccess]
+    pagination_class = DefaultPagination
+    fx_view_name = 'exported_files_data'
+    fx_default_read_only_roles = ['staff', 'instructor', 'data_researcher', 'org_course_creator_group']
+    fx_default_read_write_roles = ['staff', 'instructor', 'data_researcher', 'org_course_creator_group']
+    fx_allowed_write_methods = ['PUT']
```

Comment:
`PATCH` is the right choice
```suggestion
    fx_allowed_write_methods = ['PATCH']
```